src/sonic-config-engine/tests/test_multinpu_cfggen.py

```diff
@@ -215,17 +221,17 @@ def test_frontend_asic_ports(self):
              "Ethernet-BP12": { "admin_status": "up",  "alias": "Eth7-ASIC0",  "asic_port_name": "Eth7-ASIC0",  "description": "ASIC3:Eth1-ASIC3",  "index": "3",  "lanes": "25,26,27,28",  "mtu": "9100", "tpid": "0x8100", "pfc_asym": "off",  "role": "Int",  "speed": "40000" }})
 
     def test_frontend_asic_ports_config_db(self):
-        argument = "-m {} -n asic0 --var-json \"PORT\"".format(self.sample_graph)
+        argument = "-m {} -p {} -n asic0 --var-json \"PORT\"".format(self.sample_graph, self.port_config[0])
         output = json.loads(self.run_script(argument))
         self.assertDictEqual(output,
-                {"Ethernet0": { "admin_status": "up",  "alias": "Ethernet1/1",  "asic_port_name": "Eth0-ASIC0",  "description": "01T2:Ethernet1:config_db",  "index": "0",  "lanes": "33,34,35,36",   "mtu": "9100", "tpid": "0x8100",   "pfc_asym": "off",  "role": "Ext",  "speed": "40000", "autoneg": "on" },
-                "Ethernet4": { "admin_status": "up",  "alias": "Ethernet1/2",  "asic_port_name": "Eth1-ASIC0",  "description": "01T2:Ethernet2:config_db",  "index": "1",  "lanes": "29,30,31,32",   "mtu": "9100", "tpid": "0x8100",  "pfc_asym": "off",  "role": "Ext",  "speed": "40000", "autoneg": "on" },
-                "Ethernet8": { "admin_status": "up", "alias": "Ethernet1/3",  "asic_port_name": "Eth2-ASIC0",  "description": "Ethernet1/3:config_db",  "index": "2",  "lanes": "41,42,43,44",   "mtu": "9100", "tpid": "0x8100",  "pfc_asym": "off",  "role": "Ext",  "speed": "40000" },
```

Comment:
Why admin_status disappear? Is it expected?

Reply:
The actual output does not have admin_status